config/phobos.yml.example

```diff
@@ -74,6 +74,8 @@ consumer:
   offset_retention_time:
   # interval between heartbeats; must be less than the session window
   heartbeat_interval: 10
+  # max number of items in the fetch queue that
+  fetcher_max_queue_size: 100
```

Comment:
@mensfeld do you have any links/references for suggesting to set this to 10? I take your word for it, but it's always great to share knowledge 👍

Reply:
Only couple systems that blew up :D we've changed it in soon to be released 1.3 version of Karafka. In general: fetcher queue is not only used for commands but also for temp storing prebuffered data. With all the other options set to default, it may buffer couple GB of data upfront when there's a bigger offset lag. We've killed our systems couple times because of it being set to 100 (reached more than 2GB of memory usage). Having this setting so high by default can make phobos unpredictable for inexperienced users. They may assume the lib has memory leaks while it's just prebuffering a lot of data. ref: https://github.com/karafka/karafka/issues/342

Reply:
Thanks @mensfeld ❤️

Reply:
You can always count on me :)